openpype/modules/deadline/plugins/publish/submit_maya_deadline.py

```diff
@@ -254,7 +254,11 @@ class MayaSubmitDeadline(pyblish.api.InstancePlugin):
     use_published = True
     tile_assembler_plugin = "OpenPypeTileAssembler"
     asset_dependencies = False
+    priority = 50
+    tile_priority = 50
     limit_groups = []
+    jobInfo = None
+    pluginInfo = None
```

Comment:
Looking at the previous code these should defaulto to `{}`?

Reply:
Fixed